docs/pages/pmd/userdocs/cli_reference.md

```diff
@@ -80,6 +81,11 @@ The tool comes with a rather extensive help text, simply running with `--help`!
                             The valid values are the standard character sets of `java.nio.charset.Charset`."
                default="UTF-8"
     %}
+    {% include custom/cli_option_row.html options="--[no-]fail-on-processing-error"
+               description="Specifies whether PMD exits with non-zero status if processing errors occurred.
+                            By default PMD exits with status 5 if processing errors or violations are found.
+                            Disable this option with `--no-fail-on-processing-error` to exit with 0 instead and just write the report."
```

Comment:
I think it should be made clear here that even if the option is enabled we will write a report.